Dockerfile

```diff
@@ -28,6 +28,9 @@ RUN addgroup -S -g "${GID}" "${APP_USER}" && \
 
 WORKDIR "${CWD}"
 
+# postgresql-dev and g++ are required to build psycopg2-binary
+RUN apk add --update --no-cache postgresql-dev g++
```

Comment:
No alpine wheels? :disappointed: 

Since alpine makes it easy to declare temporary dependencies how about removing these again after building the package? We can also introduce another base layer with the psycopg2 dependency since we have a couple of services that need it. I think moving the compilation of gevent out of development builds has worked really quite well.